### ISSUE TITLE
Remove font picker from RichTextKeyboardToolbar?

### ISSUE THREAD

**Issue by nicoladaniello** [danielsaidi/RichTextKit#59]
Is there a way to remove the font family picker from `RichTextKeyboardToolbar`?

**Comment by danielsaidi** [comment]
Hi @nicoladaniello 

Not currently, but I've been planning to express the various toolbar sections in an enum to make things configurable.

I will keep this issue open until I have that in place.

**Comment by screenworker** [comment]
Hi @nicoladaniello,

just build a suitable alternative. Here is an example without a font picker:

```
public struct CustomRichTextFormatSidebar: View {
    public init(
        context: RichTextContext
    ) {
        self._context = ObservedObject(wrappedValue: context)
    }

    @ObservedObject
    private var context: RichTextContext

    private let spacing = 10.0

    public var body: some View {
        VStack(alignment: .leading, spacing: spacing) {

            SidebarSection(title: nil) {
                RichTextAlignmentPicker(selection: $context.textAlignment)
                    .pickerStyle(.segmented)
                indentButtons
            }

            SidebarSection(title: nil) {
                VStack(spacing: 4) {
                    RichTextColorPicker(color: .foreground, context: context, quickPickerColors: .curated)
                    RichTextColorPicker(color: .background, context: context, quickPickerColors: .curated)
                }
            }
            .font(.callout)
            .padding(.trailing, -8)

            Spacer()
        }
        .padding(8)
        .background(Color.white.opacity(0.05))
    }
}

private struct SidebarSection<Content: View>: View {

    let title: String?

    @ViewBuilder
    let content: () -> Content

    var body: some View {
        VStack(alignment: .leading, spacing: 10) {
            if let title {
                Text(title).font(.headline)
            }
            content()
            Divider()
        }
    }
}

private extension CustomRichTextFormatSidebar {

    @ViewBuilder
    var styleToggles: some View {
        if #available(iOS 15.0, macOS 12.0, *) {
            RichTextStyleToggleGroup(context: context)
        } else {
            RichTextStyleToggleStack(context: context)
        }
    }

    @ViewBuilder
    var indentButtons: some View {
        if #available(iOS 15.0, macOS 12.0, *) {
            RichTextActionButtonGroup(
                context: context,
                actions: [.decreaseIndent, .increaseIndent]
            )
        }
    }
}
```

**Comment by danielsaidi** [comment]
The new format toolbar and the old sidebar has new configurations in the upcoming `0.99`.
